.travis.yml

```diff
@@ -73,6 +73,11 @@ before_install:
         gcc --version;
         BAR="bar -i 30";
         BUILDOPTS="-j5 VERBOSE=1 FORCE_ASSERTIONS=1 LLVM_ASSERTIONS=1";
+        if [ "$ARCH" = "x86_64" ]; then
+            BUILDOPTS="$BUILDOPTS JULIA_CPU_TARGET=x86-64";
+        else
+            BUILDOPTS="$BUILDOPTS JULIA_CPU_TARGET=i686";
```

Comment:
this is not the correct value for 32 bit, it should be pentium4 to ensure sse2 gets used rather than x87

Reply:
It's since been changed